sample-applications/containermvc-web.yaml

```diff
@@ -13,7 +13,7 @@
 # OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
 # SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
-apiVersion: apps/v1beta1
```

Comment:
If this is deprecated for only v1.16+ and we have different yaml files for v1.16+, this change is not needed.